yarn-project/circuits.js/src/structs/complete_address.ts

```diff
@@ -34,49 +33,52 @@ export class CompleteAddress {
 
   static create(address: AztecAddress, publicKey: PublicKey, partialAddress: PartialAddress) {
     const completeAddress = new CompleteAddress(address, publicKey, partialAddress);
-    completeAddress.validate();
+    // completeAddress.validate();
```

Comment:
What about this one down here? No validation anymore?

Reply:
Linked the relevant issue here as well.

I disabled the validation because it would require too many changes. BTW CompleteAddress does not make much sense anymore with the new keys because it's no longer sufficient to verify that the contract address was derived correctly and neither it's sufficient to send a note to a user.